app/src/components/notifications/NotificationManager.vue

```diff
@@ -21,11 +28,24 @@ const { notifications } = storeToRefs(useNotificationStore());
                 leave-to-class="opacity-0"
             >
                 <LNotification
-                    v-for="notification in notifications"
+                    v-for="notification in notifications.filter((c) => c.type == 'toast')"
                     :key="notification.id"
                     :notification
                 />
             </TransitionGroup>
         </div>
     </div>
+    <div
+        aria-live="assertive"
+        class="inset-x-0 flex items-end sm:items-start"
+        v-if="type == 'banner'"
+    >
+        <div class="w-full">
+            <LNotification
+                v-for="notification in notifications.filter((c) => c.type == 'banner')"
```

Comment:
Nitpick: Especially if we split up the `NotificationManager` in two components (see my other comment), I would perhaps opt to make this a getter in the store, so you can do something like this:

```ts
const { banners } = storeToRefs(useNotificationStore());

v-for="banner in banners"
```